crossdock/client/gauntlet/behavior.go

```diff
@@ -68,116 +69,118 @@ func createGauntletT(t crossdock.T) crossdock.T {
 	}
 }
 
+// TT is the gauntlets table test struct
+type TT struct {
+	Service  string        // thrift service name; defaults to ThriftTest
+	Function string        // name of the Go function on the client
+	Details  string        // optional extra details about what this test does
+	Give     []interface{} // arguments besides thrift.Request
+
+	Want          interface{} // expected response; nil for void
+	WantError     error       // expected error
+	WantErrorLike string      // for just matching error messages
+}
+
 // Run executes the thriftgauntlet behavior.
 func Run(t crossdock.T) {
 	t = createGauntletT(t)
-	assert := crossdock.Assert(t)
 	checks := crossdock.Checks(t)
 
 	rpc := rpc.Create(t)
 
 	bytesToken := random.Bytes(10)
-	tests := []struct {
-		service  string        // thrift service name; defaults to ThriftTest
-		function string        // name of the Go function on the client
-		details  string        // optional extra details about what this test does
-		give     []interface{} // arguments besides thrift.Request
-
-		want          interface{} // expected response; nil for void
-		wantError     error       // expected error
-		wantErrorLike string      // for just matching error messages
-	}{
+	tests := []TT{
 		{
-			function: "TestBinary",
-			give:     []interface{}{bytesToken},
-			want:     bytesToken,
+			Function: "TestBinary",
+			Give:     []interface{}{bytesToken},
```

Comment:
Looks like it's necessary in order to treat all `Give`'s as the same type down below. Changing it gets me:

```
crossdock/client/gauntlet/behavior.go|95 error| cannot use bytesToken (type []byte) as type []interface {} in field value
```